### PR TITLE
clean_tmpdisk.py needs --states=RUNNING in squeue query

### PR DESCRIPTION
**Pull request by cat-bro**

### CODE REVIEW

**Comment by cat-bro** [comment]
if there were PENDING jobs in the squeue output the list unpacking "slurm_id, name, state, runtime, node = re.split('\s+', row.strip())". The script is not looking at PENDING jobs

**Review by jlqfab — APPROVED** [review]

File: roles/clean-tmpdisk/files/clean_tmpdisk.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def main():
     dry_run = args.dry_run or args.list_only # enforce dry run in the case of printing the list
     verbose = args.verbose
 
-    squeue_details = subprocess.check_output(f'squeue --format=\'{squeue_format}\'', shell=True)
+    squeue_details = subprocess.check_output(f'squeue --format=\'{squeue_format}\' --states=RUNNING', shell=True)
```
Comment (nuwang) [inline roles/clean-tmpdisk/files/clean_tmpdisk.py:47]:
```suggestion
    squeue_details = subprocess.check_output(f'squeue --format=\'{squeue_format}\' --states=RUNNING -w {worker_node}', shell=True)
```

File: roles/clean-tmpdisk/files/clean_tmpdisk.py [hunk 2]
```diff
@@ -58,7 +58,7 @@ def main():
         if not row.strip():
             continue
         slurm_id, name, state, runtime, node = re.split('\s+', row.strip())
-        if node != worker_node or state == 'PENDING':
+        if node != worker_node:
             continue
```
Comment (nuwang) [inline roles/clean-tmpdisk/files/clean_tmpdisk.py:62]:
```suggestion
```

**Review by nuwang — COMMENTED** [review]
@cat-bro Just a small optimization

**Pull request merged**